app/assets/stylesheets/application.scss

```diff
@@ -6,6 +6,7 @@
 // External libraries
 @import "bootstrap/scss/bootstrap";
 @import "font-awesome";
+@import url("https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0");
```

Comment:
move this to `app/assets/stylesheets/config/_fonts.scss`

Reply:
eh? this is the icons that I import from Google to use as our footer wor, not exactly fonts wor

Reply:
can put in _fonts.scss because they are all under // Import Google fonts